### DISCUSSION TITLE
Telegram web apps for bot

### DISCUSSION THREAD

**Discussion by fr20587** [telegraf/telegraf#1604] [Q&A] [answered]
How I can do a product menu as DurgerKingBot using telegraf?

**Comment by Bl00dyPudding** [comment]
Bot API 6.0 is not supported by Telegraf now.

**Comment by MKRhere** [comment]
API 6.0 is one day old, so we're still figuring out the changes. Since I'm out of town for 3 more days, the Telegraf update will probably land some time in the coming week.

**Reply by fr20587** [reply]
Yes I know 😅, many thanks for yours comments 👍

**Reply by varunkumarmedam** [reply]
Eagerly waiting for the web_app support

**Comment by MKRhere** [comment, marked as answer]
Addressed in #1614, will land shortly. Please follow the PR for updates.

**Comment by jakhongir-abdukhamidov** [comment]
SentWebAppMessage doesn't seem to have been added, is there any updates on this?

**Reply by MKRhere** [reply]
`SentWebAppMessage` is a type. It's returned by `bot.telegram.answerWebAppQuery`. The type itself can be imported with

```TS
import { type SentWebAppMessage } from "telegraf/types";
```

**Reply by pravosleva** [reply]
In wich version?
